docs/development_suite/monitoring/resources/jobs.md

```diff
@@ -19,9 +19,24 @@ The table presented here shows the following information:
 
 ![list_of_jobs](../img/jobs_list.png)
 
+## Deleting a Job
+
+By clicking on the **Delete button** on the last column of the Job table, the Job will be permanently deleted.
+
+A modal will ask for confirmation for the action to be performed.
+
+Any related resources, such as Pods, will be marked for deletion, and Kubernetes may delete them at any time.
```

Comment:
Could it be necessary to be more specific instead of just "may delete them at any time"?

Reply:
Yeah probably we could, maybe we add a link to the relevant k8s doc? https://kubernetes.io/docs/concepts/architecture/garbage-collection/#cascading-deletion

Reply:
```suggestion
Any related resources, such as Pods, will be marked for deletion, and Kubernetes may [cascading delete](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#cascading-deletion) them at any time.
```

What do you think?